plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+<?php
+/**
+ * SimpleProductTemplate
+ */
+
+namespace Automattic\WooCommerce\Admin\Features\ProductBlockEditor\ProductTemplates;
+
+use Automattic\WooCommerce\Admin\Features\Features;
+use Automattic\WooCommerce\Admin\BlockTemplates\BlockInterface;
+use Automattic\WooCommerce\Internal\Admin\BlockTemplates\AbstractBlockTemplate;
+
+/**
+ * Simple Product Template.
+ */
+class SimpleProductTemplate extends AbstractBlockTemplate implements ProductFormTemplateInterface {
+
+	/**
+	 * SimpleProductTemplate constructor.
+	 */
+	public function __construct() {
+		$this->add_general_group_blocks();
+		$this->add_organization_group_blocks();
+		$this->add_pricing_group_blocks();
+		$this->add_inventory_group_blocks();
+		$this->add_shipping_group_blocks();
+		if ( Features::is_enabled( 'product-variation-management' ) ) {
+			$this->add_variation_group_blocks();
+		}
+	}
+
+	/**
+	 * Get the template ID.
+	 */
+	public function get_id(): string {
+		return 'simple-product';
+	}
+
+	/**
+	 * Get the template title.
+	 */
+	public function get_title(): string {
+		return __( 'Simple Product Template', 'woocommerce' );
+	}
+
+	/**
+	 * Get the template description.
+	 */
+	public function get_description(): string {
+		return __( 'Template for the simple product form', 'woocommerce' );
+	}
+
+	/**
+	 * Get a group block by ID.
+	 *
+	 * @param string $group_id The group block ID.
+	 */
+	public function get_group_by_id( string $group_id ): ?GroupInterface {
+		return $this->get_block( $group_id );
+	}
+
+	/**
+	 * Get a section block by ID.
+	 *
+	 * @param string $section_id The section block ID.
+	 */
+	public function get_section_by_id( string $section_id ): ?SectionInterface {
+		return $this->get_block( $section_id );
+	}
+
+	/**
+	 * Get a block by ID.
+	 *
+	 * @param string $block_id The block block ID.
+	 */
+	public function get_block_by_id( string $block_id ): ?BlockInterface {
+		return $this->get_block( $block_id );
+	}
+
+	/**
+	 * Add a custom block type to this template.
+	 *
+	 * @param array $block_config The block data.
+	 */
+	public function add_group( array $block_config ): GroupInterface {
```

Comment:
`get_group_by_id`, `get_section_by_id`, `get_block_by_id`, and `add_group` all are generic methods that any "product template" would need (and they would all have the same implementation). It seems like `ProductFormTemplate` base class would be useful to define these generic methods.